### PR TITLE
Implement UI for event page

### LINKED ISSUE #90: Implement UI for event page

### PR DESCRIPTION
**Pull request by kristofferlarberg**
Implements UI for event page according to [wireframe](https://whimsical.com/gen-3-wireframes-activist-view-DgqhYzEQJtMP9naZ3BizXs).

Skärmdump 2021-03-24, 13 55 41

Fixes #90

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,13 +12,16 @@
     "start": "next start -p 80"
   },
   "nyc": {
-    "reporter": ["html"]
+    "reporter": [
+      "html"
+    ]
```
Comment (richardolsson) [inline package.json:17]: No big deal, but please be careful not to commit unrelated changes that don't mean anything. I'm guessing this change is caused by some auto-formatting feature in your editor. It should not have been committed.

File: src/pages/o/[orgId]/events/[eventId].tsx [hunk 2]
```diff
@@ -69,80 +82,80 @@ const OrgEventPage : PageWithLayout<OrgEventPageProps> = (props) => {
...
-                        value={ Date.parse(event.start_time) }
-                    />
-                    , <FormattedTime
-                        value={ Date.parse(event.start_time) }
-                    />
-                </Text>
+            <Flex alignItems="center" data-test="duration" marginBottom="size-100">
+                <Calendar aria-label="S Calender" marginEnd="size-100" size="S"/>
+                <Flex direction="column">
+                    <Text data-test="start-time">
```
Comment (richardolsson) [inline src/pages/o/[orgId]/events/[eventId].tsx:114]: The `data-test` id doesn't seem to reflect the content here. Is the test ID necessary? If not, remove it, otherwise it should be changed to correctly reflect the content.

File: src/pages/o/[orgId]/events/[eventId].tsx [hunk 3]
```diff
@@ -69,80 +82,80 @@ const OrgEventPage : PageWithLayout<OrgEventPageProps> = (props) => {
...
                     </NextLink>
                 </Link>
-            </Flex>
-            <Flex>
-                <Text marginEnd="size-50">
-                    <Msg id="pages.orgEvent.details.campaign"/>:
-                </Text>
+            </Header>
+            <Flex marginBottom="size-100">
+                <Flag aria-label="S Flag" marginEnd="size-100" size="S"/>
```
Comment (richardolsson) [inline src/pages/o/[orgId]/events/[eventId].tsx:104]: Area the aria labels required? If they are, they should probably be internationalized. Otherwise I think we should remove them in this case, where the icon is really just decoration.

**Review by richardolsson — CHANGES_REQUESTED** [review]
Great! This looks really good and I only have some very minor objections. See comments below. 🙌

File: package.json (hunk 1 above)
Comment (kristofferlarberg) [reply package.json:17]: That's very strange since I've turned off all auto-formatting, but either way I shouldn't have made it part of the commit, sorry about that.

**Review by richardolsson — APPROVED** [review]
LGTM. 👍  This goes in!

Neat!
